db/schema.rb

```diff
@@ -280,6 +309,31 @@
     t.index ["pipeline_run_id", "step_number"], name: "index_pipeline_run_stages_on_pipeline_run_id_and_step_number"
   end
 
+  create_table "pipeline_run_stages__deleted_rows", id: false, force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
```

Comment:
What's up with all the changes in schema.rb? Were these automatically added or did you edit it by hand? I think all these changes should be in a migration, no?

Reply:
Oops I accidentally committed these — will remove